### PR TITLE
Edit README file

### PR DESCRIPTION
**Pull request by Renny4Real**
Add a more descriptive purpose of the project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 ## Purpose
 
-This is the User Admin UI for the [GovWiFi][link_govwifi] project.
+This is the admin platform, a website where organisations can create and manage their GovWifi installation within their organisation.
+The Govwifi page can be found [here][link_govwifi].
```
Comment (kjdchapman) [inline README.md]: Is this link useful?

Comment (Renny4Real) [reply README.md]: Not completely sure why it added before.
I was thinking maybe we needed to direct to the live page in some way.

File: README.md [hunk 2]
```diff
@@ -1,23 +1,59 @@
...
 
-## Purpose
+This is the admin platform, a website where organisations can create and manage their GovWifi installation within their organisation.
+The private GovWifi [build repository][build-repo] contains instructions on how to build GovWifi end-to-end - the sites, services and infrastructure.
 
-This is the User Admin UI for the [GovWiFi][link_govwifi] project.
+## Overview
 
-## How to install and use
+The API contains files and test suites that describe and maintain the desired functionality of the admin platform. It is primarily a Ruby-on-Rails Project.
```
Comment (tommotaylor) [inline README.md]: This line doesn't really make much sense to me. Something better here would perhaps be a list of the features. For instance.

```
The application allows users to perform the following tasks:
- Create an admin account.
- Invite team members to their account.
- View instructions on how to setup and configure GovWifi on their local network.
- Add IP addresses of their access points to the GovWifi system.
- View logs of authentication requests to GovWifi by IP and username.
- Make support ticket requests.

The application also includes a "Super Admin" login that allows the GDS administrator to:
- View all organisations signed up to GovWifi
- See specific information on each of these organisations
- Add custom organisation names to the allowed register
```

File: README.md [hunk 3]
```diff
@@ -1,23 +1,59 @@
...
+
+The API uses some GovUK Services to implement some the desired functions, including:
```
Comment (tommotaylor) [inline README.md]: The application uses a few third party services, including:

File: README.md [hunk 4]
```diff
@@ -1,23 +1,59 @@
...
+
+* [GOV.UK Registers][registers] as a resource for Government organisations to select their names from a predetermined list of registered organisations as they create their accounts.
+
+* [GOV.UK Notify][notify] to handle sending out situational emails to users.
+
+* [GOV.UK Zendesk][zendesk] to handle forms submitted by the user within the app.
+
```
Comment (tommotaylor) [inline README.md]: The application also provides the following data for the RADIUS configuration via an S3 bucket:
- IP addresses.
- RADIUS secret keys.

File: README.md [hunk 5]
```diff
@@ -1,23 +1,59 @@
...
+## Developping
```
Comment (tommotaylor) [inline README.md]: Spelling. Developing

**Pull request merged**